SPARCore/GADArticle.m

```diff
@@ -2,6 +2,48 @@
 
 @implementation GADArticle
 
++ (void) fetchArticlesWithPublication: (NSString *)publicationId
+                                        completionHandler:(void(^_Nonnull)(NSArray<GADRemoteModel *>
+                                                                           *_Nullable models,
+                                                            NSError *_Nullable error))completion {
+    //Should we create NSDictionary parameters within each method and just request the publicationId
+    //from the front end?
+
+}
+
+- (void) populateArticleWithId: (NSString *)articleId
```

Comment:
If it loads the image from a URL into the `UIImage` property, it should not need to take an `articleId` (use `[self articleId]`) and its completion handler doesn't need to return anything except an `NSError`; though it could also return a pointer to `self` with the image property set. Ask me about this if that doesn't make sense.